docs/source/support/release-notes.rst

```diff
@@ -9,6 +9,13 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
+DAML Studio Library
```

Comment:
What's DAML studio library? My guess is that this isn't a user-facing change?

Reply:
yeah - given this was hidden before, I wouldn't add a release note.

Reply:
Oops, that should have been "DAML Standard Library". It's gone now together with the rest of the release note.